deploy/registration-service/registration-service.yaml

```diff
@@ -168,6 +154,7 @@ objects:
       namespace: ${NAMESPACE}
     spec:
       host: ''
+      path: /api # we don't want to expose anything else (eg: Prometheus metrics)
```

Comment:
this looks like a route for the registration service.
but still, do we really need to limit it? isn't it already limited/isolated using services and ports? 
apart from that - would it also allow requests on the /apis path? it's used in openshift as well.

Reply:
ohhh, that's a mistake, I thought I reverted that change! 🤦‍♂️

Reply:
removed in 7e58490